docs/build-source-code.rst

```diff
@@ -135,9 +135,18 @@ Build with the following commands:
 
 ::
 
-    cmake -DCMAKE_PREFIX_PATH="$(brew --prefix qt5)" .
+	mkdir -p build && cd build
+    cmake -DCMAKE_PREFIX_PATH=(brew --prefix qt5) -DCMAKE_LISTS_TXT_PATH="$PWD/../CMakeLists.txt" -DCMAKE_LISTS_OR_SETTINGS_JSON=CMakeListsTxtAdvanced -DCMAKE_BUILD_TYPE=Release -DCMAKE_GENERATOR=Ninja -DWITH_TESTS=TRUE ../
```

Comment:
```suggestion
    cmake -DCMAKE_PREFIX_PATH=$(brew --prefix qt5) -DCMAKE_LISTS_TXT_PATH="$PWD/../CMakeLists.txt" -DCMAKE_LISTS_OR_SETTINGS_JSON=CMakeListsTxtAdvanced -DCMAKE_BUILD_TYPE=Release -DCMAKE_GENERATOR=Ninja -DWITH_TESTS=TRUE ../
```

Reply:
Oh yeah sorry, I was using fish shell instead of bash, so that's fish syntax. Will update it to bash